src/test/java/de/samply/common/ldmclient/cql/LdmClientCqlTest.java

```diff
@@ -0,0 +1,100 @@
+package de.samply.common.ldmclient.cql;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import de.samply.common.ldmclient.LdmClientException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import org.apache.http.ProtocolVersion;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.BasicHttpEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.message.BasicStatusLine;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentMatcher;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class LdmClientCqlTest {
+
+  private static final String BASE_URL = "localhost";
+
+  @Mock
+  private CloseableHttpClient httpClient;
+
+  private LdmClientCql ldmClientCql;
+
+  @BeforeEach
+  void setUp() {
+    ldmClientCql = new LdmClientCql(httpClient, BASE_URL);
+  }
+
+  @Test
+  void testCreateMeasureReport() throws IOException, URISyntaxException, LdmClientException {
+    String measureReportUri = "/fhir/Measure/DALRE7BKVA6HY3IK";
+    CloseableHttpResponse response = mock(CloseableHttpResponse.class);
+    when(httpClient.execute(argThat(httpPostMatcher(measureReportUri + "/$evaluate-measure")))).thenReturn(response);
+    when(response.getStatusLine()).thenReturn(statusLine(201));
+    when(response.getFirstHeader("Location")).thenReturn(httpHeader());
```

Comment:
Please add a newline after in order to make the three parts of a test (preparation, execution, asserts) clear.